calcom/cal.com
# feat(app-store): add feishu calendar

## What does this PR do?

Cal.com supports Lark, the international version of Feishu. However, depending on where your tenant is, you cannot use the Lark connection.

This PR adds an app for users who are in the Feishu tenant.

Fixes #12953

## Type of change




## How should this be tested?

* Create an org on feishu.cn with an example user that has a calendar
* Create a new app at https://open.feishu.cn/  and get the app ID and secret to be used, and connect that in your dev environment
* Install app and test that it pulls in the calendar
* Confirm that events can't be booked at the same time as something already scheduled in the linked feishu calendar

(I can provide test keys/whatnot via PM if you don't wish to create an account yourself)

## Mandatory Tasks
